core/test/functional/routes/api/public_api_spec.js

```diff
@@ -188,6 +188,32 @@ describe('Public API', function () {
             });
     });
 
+    it('[deprecated] browse posts with page non matching filter', function (done) {
+        request.get(testUtils.API.getApiQuery('posts/?client_id=ghost-admin&client_secret=not_available&filter=tag:no-posts&include=tag,author'))
```

Comment:
> - included a new functional test for this case with some deprecated includes as we still support them

We have an include=author test in posts_spec right? I don't think we need this test. Let me know what you think

Reply:
This test was meant to test the filter not the includes :) Updated it in 6bc59b169e84352775834745d425a2ae8646367d to be more clear